src/app/services/subs-firestore.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
```

Comment:
The service name is weird. subs-firestore service? You can just create subscription service.

Reply:
Service name has been improved